### PR TITLE
added task created page skeleton

### PR DESCRIPTION
**Pull request by glaufer**

### CODE REVIEW

File: src/components/button.tsx [hunk 1]
```diff
@@ -36,6 +38,7 @@ export const Button: FC<ButtonProps> = ({
     onPress, 
     expandHorizontal = false, 
     disabled = false, 
+    forceForgroundStyle,
```
Comment (ellaklara) [inline src/components/button.tsx:41]:
```suggestion
    forceForegroundStyle,
```

File: src/pages/task-created.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+                <View style={styles.taskDetails}>
+                    <Center>
+                        <Text>Task details placeholder</Text>
+                    </Center>
+                </View>
+
+                <View>
+                    <View>
+                        <Button size='small' forceForgroundStyle='light' color={paperTheme.colors.important} onPress={() => {}}>
```
Comment (ellaklara) [inline src/pages/task-created.tsx:68]:
```suggestion
                        <Button size='small' forceForegroundStyle='light' color={paperTheme.colors.important} onPress={() => {}}>
```

File: src/pages/task-created.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                        <Center>
+                            <SvgIcon name='take-care'/>
+                        </Center>
+                    </View>
+                </View>
+            </View>
+            
+        </ContentPadding>
+    </View>
+}
```
Comment (ellaklara) [inline src/pages/task-created.tsx:82]: image
This is how the page looks on my phone. I think you need to look more at other pages styling to make sure it works for different devices.

File: App.tsx [hunk 4]
```diff
@@ -50,6 +51,17 @@ export default function App() {
                 ),
               }}
               component={CreateTask} />
+
```
Comment (ellaklara) [inline App.tsx:54]:
```suggestion
```

File: src/components/button.tsx [hunk 5]
```diff
@@ -17,6 +17,8 @@ interface ButtonProps {
     /** The color of the component */
     color?: string,
 
+    forceForgroundStyle?: 'light' | 'dark',
```
Comment (ellaklara) [inline src/components/button.tsx:20]:
```suggestion
    forceForegroundStyle?: 'light' | 'dark',
```

File: src/components/button.tsx [hunk 6]
```diff
@@ -49,6 +52,10 @@ export const Button: FC<ButtonProps> = ({
             }}
             labelStyle={{
                 padding: size === 'big' ? 10 : 0,
+                color: 
+                    forceForgroundStyle
```
Comment (ellaklara) [inline src/components/button.tsx:56]:
```suggestion
                    forceForegroundStyle 
```

File: src/components/button.tsx [hunk 7]
```diff
@@ -49,6 +52,10 @@ export const Button: FC<ButtonProps> = ({
             }}
             labelStyle={{
                 padding: size === 'big' ? 10 : 0,
+                color: 
+                    forceForgroundStyle 
+                    ? (forceForgroundStyle === 'light' ? '#fff' : '#000')
```
Comment (ellaklara) [inline src/components/button.tsx:57]:
```suggestion
                    ? (forceForegroundStyle === 'light' ? '#fff' : '#000')
```

File: src/pages/task-created.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (glaufer) [reply src/pages/task-created.tsx]: It seems like the easiest and most logical solution will be to go back to Receive Help: Settings page (where you initially create a new task) and edit the information there.

[the remark this answers is not in the document]
Comment (AdamJonsson) [reply src/pages/task-created.tsx]: New issue here:
https://github.com/DM2518-TakeCare/TakeCare/issues/71

**Pull request merged**